static/sass/_snapcraft_p-tabs.scss

```diff
@@ -8,6 +8,21 @@
     &__list {
       padding-right: 0;
       width: auto;
+      margin-bottom: -1px !important;
+    }
+
+    & .is-bordered {
```

Comment:
This seems too hacky to me. We are using Vanilla built in class name `is-bordered`, but we disable what it originally does to add our own behaviour. Maybe we just should use custom style.